modules/viewdeosDXBidAdapter.js

```diff
@@ -0,0 +1,243 @@
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {VIDEO, BANNER} from '../src/mediaTypes';
+import {Renderer} from '../src/Renderer';
+import findIndex from 'core-js/library/fn/array/find-index';
+
+const URL = '//hb.sync.viewdeos.com/auction/';
+const OUTSTREAM_SRC = '//player.sync.viewdeos.com/outstream-unit/2.01/outstream.min.js';
+const BIDDER_CODE = 'viewdeosDX';
+const OUTSTREAM = 'outstream';
+const DISPLAY = 'display';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['viewdeos'],
+  supportedMediaTypes: [VIDEO, BANNER],
+  isBidRequestValid: function (bid) {
+    return bid && bid.params && bid.params.aid;
+  },
+  getUserSyncs: function (syncOptions, serverResponses) {
+    var syncs = [];
```

Comment:
since `syncs` is not reassigned, it can be declared with `const` rather than `var`